sql/sql.y

```diff
@@ -74,11 +74,16 @@
 	type trainClause struct {
 		estimator string
 		attrs     attrs
-		columns   exprlist
+		ccs   	  []columnsClause
                 label     string
 		save      string
 	}
 
+	type columnsClause struct {
+		name 	string
```

Comment:
If `name` is the identifier after `FOR`, I think it would be better named `target` or `purpose` than `name`.

Reply:
Ok